src/components/Menu.vue

```diff
@@ -1,46 +1,51 @@
 <template>
-  <div :class="suit()" v-if="state">
```

Comment:
Does this fit is.css?

Reply:
Yes it fits since the full DOM widget is under the `slot`. It's just that we have now an extra (useless) `div` on top of the widget.